service/templates/task.html

```diff
@@ -470,15 +470,15 @@ <h4 class="text-center text-muted">No test results to show</h4>
                           </td>
                           <td class="col-lg-3">
                             <div class="url-link nowrap" id="html-link">
-                                <a ng-href="[[getURL(test.test_result, false)]]" ng-hide="[[hideURL(test.test_result, false)]]">
+                                <a ng-href="[[test.test_result.url]]">
                                   HTML
                                 </a>
                               </div>
                               <div id="separator">
-                                <b ng-show="[[hasBothURL(test)]]">&middot;</b>
```

Comment:
Is it alright to assume that both the raw and HTML URLs will always be present for a task (including display tasks)?

Reply:
Well, for test logs in cedar we don't know if they exist until we actually hit the route.